config/locales/client.en.yml

```diff
@@ -589,6 +589,8 @@ en:
           one: "You have pending reviewables from disabled plugin:"
           other: "You have pending reviewables from disabled plugins:"
         instruction: "They cannot be properly displayed until you enable the relevant plugin. Please enable the plugin and refresh the page. Alternatively, you can ignore them. <a href='%{url}' target='_blank'>Learn more...</a>"
+        reviewable_unknown_source: "%{type} (unknown plugin)"
+        reviewable_known_source: "%{type} (from the <tt>%{source}</tt> plugin)"
```

Comment:
Those variable names aren't really self-explanatory. As a translator, I'd have trouble knowing what they are.

`type` is `reviewableType`?
`source` is `pluginName`?

And instead of the deprecated `<tt>` tag, maybe single quotes would be enough?

Reply:
Thanks for the suggestions, @gschlager! I've made those changes. 🙂